src/6model/containers.c

```diff
@@ -114,6 +114,7 @@ static const MVMContainerSpec code_pair_spec = {
     code_pair_fetch_s,
     code_pair_store,
     code_pair_store_i,
+    code_pair_store_i,
     code_pair_store_n,
```

Comment:
It's not clear from this commit why we can get away with using `code_pair_store_i` (ie no need for `code_pair_store_u`) and similarly no need for for `value_desc_cont_store_u` when we *do* need `native_ref_store_u`

Reply:
Excellent question! The short answer is: we don't.

The slightly longer one is that the tests we have do let us get away with this, but we really shouldn't. I was thinking that I'll leave this alone for now. The branches are already huge and I'd like to see them debugged and merged first before I (or someone else) start attacking the remaining uint issues. The compiler does warn about this particular sloppiness, so it cannot fall through the cracks anyway.

Of course, if as a reviewer you disagree, I can change my strategy.

Reply:
> The compiler does warn about this particular sloppiness, so it cannot fall through the cracks anyway.

For the version of gcc that I'm using (with some or other flags) it doesn't warn for me.

I guess I'd be happier simply with a comment near line 116 roughly as you wrote. Maybe something like

*FIXME - to address all the uint issues we also need code_pair_store_u and value_desc_cont_store_u. However, our current tests don't expose this, and we don't need them for the rebootstrapping. Hence leave this code (and subtle bugs) as-is until we have the large and intertangled changes done and stable, to reduce the size and risk of those changes.*

Also, if you're rebasing the branch again, I think I'd prefer it if the one "word" commit messages such as `MVM_repr_bind_pos_u` became `Add MVM_repr_bind_pos_u`. Although thinking a bit more (but sorry, not fully baked thoughts) I *think* that it might be better to merge at least some of the commits like 92f24b80e9c4e1133ff36e893344ef28da2909f7 which merely *add* API calls, but don't change any code to call them.

(Keeping he other commits that add members to structs, and change the initialisation code in lockstep as individual commits makes sense to me, as those sort of changes are error prone, and hence having a discrete step to bisect is useful. Likewise the commits that add code to switch statements - they are also useful as individual bisects. I checked the former manually and didn't see any errors. I didn't check the latter, but "works on my machine" so I'll pretend that I did a good job of reviewing them)